.travis.yml

```diff
@@ -3,6 +3,15 @@ sudo: false
 
 matrix:
   include:
+    - name: "miri"
+      rust: nightly
+      install:
+        - rustup component add rust-src
+        - cargo +nightly install --force --git https://github.com/rust-lang/miri miri
+        - cargo install xargo
```

Comment:
This (and the `rustup component add`) should be replaced by `cargo +nightly miri setup`.